launcher/ui/pages/instance/OtherLogsPage.cpp

```diff
@@ -243,15 +252,15 @@ void OtherLogsPage::on_btnClean_clicked()
         return;
     }
     QMessageBox *messageBox = new QMessageBox(this);
-    messageBox->setWindowTitle(tr("Clean up"));
+    messageBox->setWindowTitle(tr("CAREFUL!"));
```

Comment:
```suggestion
    messageBox->setWindowTitle(tr("Confirm deletion"));
```

Reply:
Maybe Confirm Clean-up/Cleanup? (clean-up instead of clean up to turn it into a noun, like anyone cares)

Reply:
sure, cleanup (i think without hyphen looks a bit nicer, but i don't mind either way) is fine!